spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -219,10 +219,6 @@
     context 'when the user enters a valid OTP' do
       before do
         sign_in_before_2fa
-        form = OtpVerificationForm.new(subject.current_user, nil)
-        result = FormResponse.new(success: true, serialize_error_details_only: {})
-        expect(form).to receive(:submit).and_return(result)
-        expect(subject).to receive(:otp_verification_form).and_return(form)
```

Comment:
Removing these lines had no effect on the tests, but they were making it more difficult for me to test with a leading #-sign